packages/odyssey/src/scss/components/_tab.scss

```diff
@@ -12,13 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.ods-tabs {
-  --ods-tabs-indicator-width: 0;
-  --ods-tabs-indicator-pos-x: 0;
-}
```

Comment:
Just a reminder - because you're changing the behavior of existing components - you'll need to call out this in our CHANGELOG as a potential breaking change. Because it is buried into this PR, it's going to be difficult to point developers to the change.

Since #731 was believed to unblock deploys (which appears didn't work) - can we drop that commit from this working tree and merge it into `develop` after?

Reply:
@jmelberg-okta merging #731 did work the workflow successfully deployed for the commit: `fac3e75`

The reason it no longer works is due to an asset not being included which is required at build time. It is in the content PR (#732)

Once merged, the build will successfully complete once more. It can be argued that that asset should be part of this package, as it is required for building. I will make a note of this and do it in a follow-up PR.